golang/cosmos/app/app.go

```diff
@@ -837,20 +831,39 @@ func NewAgoricApp(
 	return app
 }
 
+var upgradeNamesOfThisVersion = map[string]bool{
+	"agoric-upgrade-14":         true,
+	"agorictest-upgrade-14":     true,
+	"agorictest-upgrade-14-rc1": true,
```

Comment:
What about "agoric-upgrade-14-rc1"?

Reply:
doesn't exist

Reply:
I feel we should name this something else than `-rc1` to avoid confusions. Maybe `-2` ?